src/types.jl

```diff
@@ -35,76 +31,73 @@ struct FontExtent{T}
     scale::Vec{2, T}
 end
 
-hadvance(ext::FontExtent) = ext.advance[1]
-inkwidth(ext::FontExtent) = ext.scale[1]
-inkheight(ext::FontExtent) = ext.scale[2]
-hbearing_ori_to_left(ext::FontExtent) = ext.horizontal_bearing[1]
-hbearing_ori_to_top(ext::FontExtent) = ext.horizontal_bearing[2]
-leftinkbound(ext::FontExtent) = hbearing_ori_to_left(ext)
-rightinkbound(ext::FontExtent) = leftinkbound(ext) + inkwidth(ext)
-bottominkbound(ext::FontExtent) = hbearing_ori_to_top(ext) - inkheight(ext)
-topinkbound(ext::FontExtent) = hbearing_ori_to_top(ext)
+@inline hadvance(ext::FontExtent) = ext.advance[1]
```

Comment:
why adding inline here? I'd leave this up to julia...

Reply:
Fixed.